packages/docusaurus-theme-translations/locales/is/theme-common.json

```
@@ -0,0 +1,74 @@
+  "theme.admonition.tip": "ábending",
+  "theme.admonition.warning": "aðvörun",
+  "theme.blog.archive.description": "Skjalasafn",
+  "theme.blog.archive.title": "Skjalasafn",
+  "theme.blog.paginator.navAriaLabel": "Blogg listsíðu yfirlit",
+  "theme.blog.paginator.newerEntries": "Nýrri færslur",
+  "theme.blog.paginator.olderEntries": "Eldri færslur",
+  "theme.blog.post.paginator.navAriaLabel": "Blogg færslu síðu yfirlit",
+  "theme.blog.post.paginator.newerPost": "Nýrri færsla",
+  "theme.blog.post.paginator.olderPost": "Eldri færsla",
+  "theme.blog.post.plurals": "Ein færsla|{count} færslur",
+  "theme.blog.post.readMore": "Lesa Meira",
+  "theme.blog.post.readMoreLabel": "Lesa meira um {title}",
+  "theme.blog.post.readingTime.plurals": "Einnar mínútu lestur|{readingTime} mínútu lestur",
+  "theme.blog.sidebar.navAriaLabel": "Yfirlit fyrir nýlegar blogg færslur",
+  "theme.blog.tagTitle": "{nPosts} merkt með \"{tagName}\"",
+  "theme.colorToggle.ariaLabel": "Skiptu á milli rökkur stillingar og bjartar stillingar (núna {mode})",
+  "theme.colorToggle.ariaLabel.mode.dark": "rökkur stilling",
+  "theme.colorToggle.ariaLabel.mode.light": "björt stilling",
+  "theme.common.editThisPage": "Breyttu þessari síðu",
+  "theme.common.headingLinkTitle": "Beinn hlekkur að {heading}",
+  "theme.common.skipToMainContent": "Hoppa yfir á aðal efni",
+  "theme.docs.DocCard.categoryDescription": "{count} atriði",
+  "theme.docs.breadcrumbs.home": "Heimasíða",
+  "theme.docs.breadcrumbs.navAriaLabel": "Brauðteningar",
+  "theme.docs.paginator.navAriaLabel": "Skjala síður",
+  "theme.docs.paginator.next": "Næsta",
+  "theme.docs.paginator.previous": "Fyrra",
+  "theme.docs.sidebar.closeSidebarButtonAriaLabel": "Loka yfirlits borða",
+  "theme.docs.sidebar.collapseButtonAriaLabel": "Loka spássíu",
+  "theme.docs.sidebar.collapseButtonTitle": "Loka spássíu",
+  "theme.docs.sidebar.expandButtonAriaLabel": "Opna spássíu",
+  "theme.docs.sidebar.expandButtonTitle": "Opna spássíu",
+  "theme.docs.sidebar.navAriaLabel": "Skjala spássía",
+  "theme.docs.sidebar.toggleSidebarButtonAriaLabel": "Víxsla yfirlits borða",
+  "theme.docs.tagDocListPageTitle": "{nDocsTagged} með \"{tagName}\"",
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "Eitt skjal merkt|{count} skjöl merkt",
+  "theme.docs.versionBadge.label": "Útgáfa: {versionLabel}",
+  "theme.docs.versions.latestVersionLinkLabel": "nýjasta útgáfa",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Fyrir uppfærða skjölun, skoðið {latestVersionLink} ({versionLabel}).",
+  "theme.docs.versions.unmaintainedVersionLabel": "Þetta er skjölun fyrir {siteTitle} {versionLabel}, sem ekki er lengur viðhaldið.",
+  "theme.docs.versions.unreleasedVersionLabel": "Þetta er óútgefin skjölun fyrir {siteTitle} {versionLabel} útgáfa.",
+  "theme.lastUpdated.atDate": " þann {date}",
+  "theme.lastUpdated.byUser": " af {user}",
+  "theme.lastUpdated.lastUpdatedAtBy": "Seinast uppfært {atDate}{byUser}",
```

```suggestion
  "theme.lastUpdated.atDate": " þann {date}",
  "theme.lastUpdated.byUser": " af {user}",
  "theme.lastUpdated.lastUpdatedAtBy": "Seinast uppfært{atDate}{byUser}",
```

The "space" separators are already included in the 2 strings above